src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -415,6 +418,10 @@ public String getCurrentSubnetId() {
     public boolean getAssociatePublicIp() {
         return associatePublicIp;
     }
+    
+    public String getMinUpHours() {
+ 		return minUpHours;
+ 	}
```

Comment:
Mind the indentation

Reply:
hmmm , not sure why my eclipse is messing up formatting only for this line. Let me use text editor in next commit